build.washingtonpost.com/components/Layout/Components/Sidebar.js

```diff
@@ -201,7 +201,7 @@ export default function Sidebar({ navigation, setMobileMenu }) {
                   <div>{item.frontMatter.data.title}</div>
                   <Change type="ComingSoon">Coming soon</Change>
                 </CustomLink>
-              ) : item.frontMatter.data.status == "Draft" ? (
+              ) : item.frontMatter.data.status == "Alpha" ? (
```

Comment:
⛏️ Can we abstract this look up?
```suggestion
              ) : StatusHelpers.isAlpha(item.frontMatter.data.status) ? (
```